app/components/work_show_info_component.html.erb

```diff
@@ -197,3 +197,32 @@
   </ul>
 <% end %>
 
+<% if more_like_this_works.present? %>
+  <h2 class="attribute-sub-head">More Like This
+      <a class="btn btn-primary" data-toggle="collapse" href="#moreLikeThisJson" role="button" aria-expanded="false" aria-controls="moreLikeThisJson">
+    Justify choices
+  </a></h2>
+  <ul class="related-items show-sub-head-body list-unstyled">
+    <% more_like_this_works[0..2].each do |work| %>
+      <li class="related-work">
+        <div class="related-work-thumb">
+          <%= link_to work_path(work.friendlier_id) do %>
+            <%= render ThumbComponent.new(work.leaf_representative, thumb_size: :standard, lazy: true) %>
+          <% end %>
+        </div>
+        <div class="related-work-text">
+          <%= link_to work_path(work.friendlier_id) do %>
+            <%= work.title %>
+          <% end %>
+        </div>
+      </li>
+    <% end %>
+  </ul>
+  <p>NOTE: the more-like-this section is just a demo. Presentation will change after we evaluate this feature.</p>
+<% end %>
```

Comment:
We should DRY up the identical display of works for related items and more like this. In each case we have an actual work, can we make a component that takes a Work as an argument, and displays it in this fashion?

(In the future, I imagine enhancing this section so instead of a title, it may also shows other metadata more similar to a search results listing. Not part of this PR, but if we have a DRY component it's the place to do it).